persistent/ChangeLog.md

```diff
@@ -2,6 +2,8 @@
 
 ## 2.12.1.0
 
+* [#1231](https://github.com/yesodweb/persistent/pull/1231)
+    * Simplify Line type in Quasi module, always use NonEmpty 
```

Comment:
`2.12.1.0` has already been released, this is going to get bundled in with 2.12.1.1

Reply:
No probs I will shift this up :+1: